packages/web/src/pages/dashboard-page/DashboardPage.tsx

```diff
@@ -168,76 +142,84 @@ export const ArtistDashboardPage = () => {
+      <Suspense fallback={<div className={styles.chartFallback} />}>
+        <TotalPlaysChart
+          data={chartData}
+          theme={theme}
+          tracks={chartTracks}
+          selectedTrack={selectedTrack}
+          onSetYearOption={onSetYearOption}
+          onSetTrackOption={setSelectedTrack}
+          accountCreatedAt={account.created_at}
+        />
+      </Suspense>
+    )
+  }, [account, theme, listenData, onSetYearOption, selectedTrack, tracks])
+
+  const renderStats = useCallback(() => {
+    if (!account) return null
+
+    const statTiles: ReactNode[] = []
+    each(stats, (stat, title) =>
+      statTiles.push(<StatTile key={title} title={title} value={stat} />)
+    )
+
+    return <div className={styles.statsContainer}>{statTiles}</div>
+  }, [account, stats])
+
+  const renderTable = useCallback(() => {
+    const trackCount = account?.track_count || 0
+    if (!account || !(trackCount > 0)) return null
+
+    const statTiles: ReactNode[] = []
+    each(stats, (stat, title) =>
+      statTiles.push(<StatTile key={title} title={title} value={stat} />)
+    )
+
     const dataSource = formatMetadata(tracks)
     return (
-      <>
-        <div className={styles.sectionContainer}>
-          <Suspense fallback={<div className={styles.chartFallback} />}>
-            <TotalPlaysChart
-              data={chartData}
-              theme={theme}
-              tracks={chartTracks}
-              selectedTrack={selectedTrack}
-              onSetYearOption={onSetYearOption}
-              onSetTrackOption={setSelectedTrack}
-              accountCreatedAt={account.created_at}
-            />
-          </Suspense>
-        </div>
-        <div className={cn(styles.sectionContainer, styles.statsContainer)}>
-          {statTiles}
-        </div>
-        <div className={styles.tracksTableWrapper}>
-          <TracksTableContainer
-            onClickRow={onClickRow}
-            dataSource={dataSource}
-            account={account}
-          />
-        </div>
-      </>
+      <div className={styles.tracksTableWrapper}>
+        <TracksTableContainer
+          onClickRow={onClickRow}
+          dataSource={dataSource}
+          account={account}
+        />
+      </div>
     )
-  }, [
-    account,
-    theme,
-    listenData,
-    onClickRow,
-    onSetYearOption,
-    selectedTrack,
-    stats,
-    tracks
-  ])
+  }, [account, onClickRow, stats, tracks])
 
   return (
     <Page
-      title='Dashboard'
-      description='View important stats like plays, reposts, and more.'
+      title={messages.title}
+      description={messages.description}
       contentClassName={styles.pageContainer}
       header={header}
     >
-      {!account || !balance || !listenData || status === Status.LOADING ? (
+      {!account ||
+      balance === null ||
+      !listenData ||
+      status === Status.LOADING ? (
         <LoadingSpinner className={styles.spinner} />
       ) : (
         <>
-          {isUSDCEnabled ? (
```

Comment:
noticing the new code doesn't use the feature flag. just flagging in case

Reply:
I just deleted the old stuff b/c the layout is too different now